ldi-orchestrator/ldio-connectors/ldio-rml-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioRmlAdapterAutoConfig.java

```diff
@@ -11,7 +11,7 @@
 public class LdioRmlAdapterAutoConfig {
 
 	@SuppressWarnings("java:S6830")
-	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldi.RmlAdapter")
+	@Bean("Ldio:RmlAdapter")
 	public LdioAdapterConfigurator ldiHttpOutConfigurator() {
```

Comment:
This is more outside of the scope of this story, but I would expect this method name to be more like `ldioAdapterConfigurator` or `ldioRmlAdapterProcessorConfigurator`